### PR TITLE
docs: use autogenerated methods to get names from admin samples

### PR DESCRIPTION
**Pull request by rahul2393**
* Updated admin samples to remove usage of handwritten instance and database classes.

### CODE REVIEW

File: samples/samples/backup_sample.py [hunk 1]
```diff
@@ -35,23 +35,25 @@ def create_backup(instance_id, database_id, backup_id, version_time):
         backup as backup_pb
 
     spanner_client = spanner.Client()
-    instance = spanner_client.instance(instance_id)
-    database = instance.database(database_id)
+    instance_admin_api = spanner_client.instance_admin_api
```
Comment (arpan14) [inline samples/samples/backup_sample.py:38]: Do we really need both `instance_admin_api` and `database_admin_api`?

Its a bit weird that customers need to create both these clients even though all they need is an utility method - `instance_path`.

File: samples/samples/backup_sample.py [hunk 2]
```diff
@@ -465,11 +502,14 @@ def update_backup(instance_id, backup_id):
         backup as backup_pb
 
     spanner_client = spanner.Client()
-    instance = spanner_client.instance(instance_id)
+    instance_admin_api = spanner_client.instance_admin_api
 
     backup = spanner_client.database_admin_api.get_backup(
         backup_pb.GetBackupRequest(
-            name="{}/backups/{}".format(instance.name, backup_id)
+            name="{}/backups/{}".format(
```
Comment (arpan14) [inline samples/samples/backup_sample.py:509]: use backup_path if available.

File: samples/samples/pg_snippets.py [hunk 3]
```diff
@@ -1681,6 +1690,9 @@ def insert_customers(transaction):
             )
         )
 
+    instance = spanner_client.instance(instance_id)
```
Comment (arpan14) [inline samples/samples/pg_snippets.py:1693]: why do we need this handwritten objects?

File: samples/samples/snippets.py [hunk 4]
```diff
@@ -2862,6 +2903,9 @@ def insert_customers(transaction):
             )
         )
 
+    instance = spanner_client.instance(instance_id)
```
Comment (arpan14) [inline samples/samples/snippets.py:2906]: where are these handwritten objects used? If not needed, can we perform a search in the entire folder and remove the references of handwritten objects?

Comment (rahul2393) [reply samples/samples/snippets.py:2906]: We need this to do data client inserts.

File: samples/samples/pg_snippets.py (hunk 3 above)
Comment (rahul2393) [reply samples/samples/pg_snippets.py:1693]: We need this to do data client inserts.

**Pull request merged**